cloudinary-devs/gatsby-transformer-cloudinary
# feat: support for gatsby-plugin-image

We need help testing this PR, especially on existing larger projects 🙏 

Install the beta version:
- `yarn add gatsby-transformer-cloudinary@beta` or `npm install gatsby-transformer-cloudinary@beta`

Test that `gatsby-image` still works:
- Run the project as is and confirm it still works as expected
- Add a `✅ gatsby-image` comment below if it does,
- or a `⛔️ gatsby-image` with more information if it does not.
  
Test that `gatsby-plugin-image` works:
- Install version 1.14.2 that works with Gatsby v3 
  - `yarn add gatsby-plugin-image@1.14.2`or `npm install gatsby-plugin-image@1.14.2`
- Migrate to the new GatsbyImage component and confirm the images you see have not changed
- Add a `✅ gatsby-plugin-image` comment below if it does,
- or a `⛔️ gatsby-plugin-image` with more information if it does not.

Migration can be done one query at the time, no need to migrate all at once as the `fluid` and `fixed` resolvers are still there together with the new `gatsbyImageData`. [Check out the examples](https://github.com/raae/gatsby-transformer-cloudinary/tree/gatsby-plugin-image/sites/site-theme/src/examples) for gatsby-image and gatsby-plugin-images side by side.
